webhooks/2025-04-15.yml

```diff
@@ -2480,13 +2479,19 @@ components:
       required:
         - id
         - image_url
+        - user_id
+        - first_name
```

Comment:
Is `first_name` or are any of the following attributes `nullable`? I know that `first_name` and `last_name` are not required on Clerk User's by default, and Stripe cannot guarantee that every transaction has a `name`.

Reply:
They are not nullable in our webhooks right now, and we made the decision to not break the webhooks by making them nullable until we have better tooling for managing that. Unset `first_name` and `last_name` fields will show up as `""`, as they have been.